MultiSource/Benchmarks/Ptrdist/yacr2/vcg.c

```diff
@@ -286,6 +286,7 @@ SCCofVCG(_Array_ptr<nodeVCGType> VCG : count(channelNets + 1),
     /*
      * Identify all SCC.
      */
+    ulong originalTotalSCC = totalSCC;
```

Comment:
This is confusing to follow.  I think you should rename the parameter perSCC and introduce perSCC earlier.  Then you can introduce a new local variable with the correct bounds there.

Reply:
Should the perSCC parameter's bounds no longer involve totalSCC? If the bounds of perSCC use the value of totalSCC, I'm not sure how to avoid the compiler error due to losing the value of totalSCC on line 290 without introducing the originalTotalSCC variable.

Reply:
I am recommending you introduce a  new variable that replaces the parameter perSCC.   Then introduce perSCC as a new local variable initialized to the new variable, with bounds given by an originalTotalSCC variable. 

In general, this benchmark is highly problematic: we shouldn't have parameters or local variables with bounds given by totalSCC.   Those bounds would be invalidated by any function call.  Fixing that is outside the scope of this PR.